pkg/k8shandler/cluster.go

```diff
@@ -274,3 +306,89 @@ func (cState *ClusterState) amendPods(dpl *v1alpha1.Elasticsearch) error {
 	}
 	return nil
 }
+
+func updateClusterSettings(dpl *v1alpha1.Elasticsearch) error {
+
```

Comment:
I'm not sure... on one hand it makes sense because its part of `cluster` logic. I think for now lets keep it in here and we can evaluate doing a refactor later since I feel like a lot of logic in the operator can be better organized.

Reply:
ok, I'll keep it where it is now